edit_course/course_forms.py

```diff
@@ -206,9 +207,27 @@ class CloneInstanceForm(forms.Form):
         initial=True,
     )
 
+    def set_sis_selector(self) -> None:
+        sis: StudentInfoSystem = get_sis_configuration()
+        if not sis:
+            # Student Info System not configured
+            return
+        
+        instances = sis.get_instances(self.instance.course.code)
+
+        # If there are no SIS instances by this course code, don't show menu
+        if len(instances) > 0:
+            options = [('none', 'None')] + instances
```

Comment:
The `'None'` text visible in the UI could be just `'---------'` so that there is no need to translate it. Or you can translate "None".
```suggestion
            options = [('none', '---------')] + instances
```